boot/configure.ml

```diff
@@ -101,6 +101,11 @@ let () =
       , " Enable ocamlformat dev-tool, allows 'dune fmt' to build ocamlformat and use \
          it, independently from the project depenedencies .\n\
         \      This flag is experimental and shouldn't be relied on by packagers." )
+    ; ( "--bin-dev-tools"
+      , toggle "bin_dev_tools"
+      , " Enable ocamlformat dev-tool, allows 'dune fmt' to build ocamlformat and use \
+         it, independently from the project depenedencies .\n\
```

Comment:
I don't think the description is correct; this flag seems to be about implicitly adding the binary package opam repository.